client/public/index.html

```diff
@@ -9,6 +9,10 @@
       name="description"
       content="Web site created using create-react-app"
```

Comment:
You can enter the name of your app as title and a short description as content ;)

Reply:
Thank you, i missed to change this.